src/current/v23.1/log-sql-statistics-to-datadog.md

```diff
@@ -0,0 +1,90 @@
+
+CockroachDB supports a built-in integration with Datadog which sends query events as logs via the [Datadog HTTP API](https://docs.datadoghq.com/api/latest/logs/). This integration is the recommended path to achieve high throughput data ingestion, which will in turn provide more query events for greater workload observability.
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+## Step 1. Create a Datadog API key
+
+1. In Datadog, navigate to [**Organization Settings > API keys**](https://app.datadoghq.com/organization-settings/api-keys).
+1. Follow the steps in the Datadog documentation on how to [add an API key](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
+1. Copy the newly created API key to be used in Step 2.
+
+## Step 2. Configure an HTTP network collector for Datadog
+
+Configure an [HTTP network collector]({% link {{ page.version.version }}/configure-logs.md %}#output-to-http-network-collectors) by creating or modifying the [`logs.yaml` file]({% link {{ page.version.version }}/configure-logs.md %}#yaml-payload).
+
+{{site.data.alerts.callout_danger}}
+Given the [volume of `sampled_query` events](#step-3-configure-cockroachdb-to-emit-query-events), do not write `sampled_query` events to disk, or [`file-groups`]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files). Writing a high volume of `sampled_query` events to a file group will unnecessarily consume cluster resources and impact workload performance. 
+
+`sampled_query` events are a type of [telemetry event]({% link {{ page.version.version }}/eventlog.md %}#telemetry-events). The [default logging configuration]({% link {{ page.version.version }}/configure-logs.md %}#default-logging-configuration) includes a `telemetry` file-group with `channels` set to `[TELEMETRY]`.
+
+To avoid writing `sampled_query` events (and other telemetry events) to disk and disable file creation of the `telemetry` file-group:
+
+- Override the default logging configuration for the `telemetry` file-group by setting `channels` to empty square brackets `[]`.
+{{site.data.alerts.end}}
+
+In this `logs.yaml` example:
+
+   1. To send sampled query events directly to Datadog without writing events to disk, override telemetry default configuration by setting `file-groups: telemetry: channels:` to `[]`.
+   1. To connect to Datadog, replace `{DATADOG API KEY}` with the value you copied in Step 1.
+   1. To control the ingestion and potential drop rate for sampled query events, configure the following `buffering` values depending on your workload:
+
+   - `max-staleness`: The maximum time a log message will wait in the buffer before a flush is triggered. Set to `0` to disable flushing based on elapsed time. Default: `5s`
```

Comment:
These could be indented for readability

Reply:
I was unable to indent in markdown, so I could not update.